wordpress-mobile/WordPress-iOS
# Issues/5205 die svg die

Fixes #5205

To test:
View a feed where some of the posts contain a single .svg image, and where they contain a wpcom smiley.  Make sure that none of the .svg images are picked up.

Needs review: @jleandroperez
